lib/debug.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+const { debuglog } = require('util')
+
+module.exports = debuglog
+module.exports.default = module.exports
+module.exports.back = module.exports.default('nock:back');
+module.exports.common = module.exports.default('nock:common');
+module.exports.intercept = module.exports.default('nock:intercept');
+module.exports.request_overrider = module.exports.default('nock:request_overrider');
+module.exports.playback_interceptor = module.exports.default('nock:playback_interceptor');
+module.exports.recorder = module.exports.default('nock:recorder');
+module.exports.socket = module.exports.default('nock:socket');
+module.exports.scopeDebuglog = (namespace) => module.exports(`nock:scope:${  namespace}`);
```

Comment:
TIL debuglog! I had no idea and it's been around for so long! I'm down replacing the debug module for sure if we can

Reply:
Same!